content/2016.12.how-to-develop-a-webapp-with-Kinto.md

```diff
@@ -0,0 +1,244 @@
+Title: How to develop a WebApp with Kinto
+Date: 2016-12-01 10:20
+Slug: develop-a-chat-webapp-with-kinto
+Authors: Enguerran Colson
+Summary: An example of how Kinto handles a webapp's data storage.
+
+## Our goal
+
+Hi folks, today, we are gonna create a simple WebApp that allows visitors to chat in a room accessible from a public URL.
+
+![Kinto Chat mobile view](images/kintochat-fullscreen.png)
+{.image}
```

Comment:
TIL : I did not know `{.image}` to apply `class="image"` onto the generated image tag